### DISCUSSION TITLE
[Request]: disable the 'use' (throwing / spawning) of chickens eggs in someone's claim

### DISCUSSION THREAD

**Discussion by TheSenseiNL** [GriefPrevention/GriefPrevention#201] [General]
Feature Request:

Hi, i already asked this on Spigot and you told me that yo would look in to the idea, but asked me to post the 'feature request' here so it won't be forgotten. so here it is:

Please,  add the feature to disable egg bombing on a players claim, and please disable it on default.

I run a shop and everytime player come in to the shop they egg-bomb it, throwing a full inventory of chicken eggs in to my shop spawning chicken.
I find that funny and can take a joke, but not when i have to kill around 2000 chickens (no joke) per 24 hours.

Please add this feature to Grief Prevention

(p.s. i think it's indeed possible to track the 'egg' source and to disallow it. but you'll probably know more about that then me).

**Comment by TheSenseiNL** [comment]
Any news yet?   (i know you prob got more to do then only coding the plugin  :p )

**Comment by RoboMWM** [comment]
Wondering if this behavior is similar/can be abstracted to any other vanilla mechanic. Only one I can think of is mob spawn eggs, and even that's a bit of a stretch.
